docs/interfaces/protobuf/reference.md

```diff
@@ -154,66 +177,138 @@ q).protobufkdb.listImportedMessageTypes[]
 `ScalarExampleDynamic`RepeatedExampleDynamic`SubMessageExampleDynamic`MapExam..
 ```
 
+## `displayMessageSchema`
 
-## `loadMessage`
+_Display the Proto schema of a specified message_
 
 ```txt
-.protobufkdb.loadMessage[message_type;file_name]
+.protobufkdb.displayMessageSchema[message_type]
+```
+
+Where `message_type` is the name of a message type as a string, 
+
+1.  prints schema format to stdout 
+1.  returns generic null
+
+`message_type` must match a message name in its `.proto` definition.
+
+??? warning "For use only in debugging"
+
+    The schema is generated by the libprotobuf `DebugString()` functionality and displayed on stdout to preserve formatting and indentation.
+
+```q
+q).protobufkdb.displayMessageSchema[`ScalarExampleDynamic]
+message ScalarExampleDynamic {
+  int32 scalar_int32 = 1;
+  double scalar_double = 2;
+  string scalar_string = 3;
+}
+```
+
+## `getMessageFields`
+
+_Get the list of message fields_
+
+```txt
+.protobufkdb.getMessageFields[message_type]
+```
+
+Where `message_type` is a message type (string or symbol) matching a message name in the `.proto` definition
```

Comment:
Itemize?
Ref) https://code.kx.com/q/interfaces/kafka/reference/#kfksubscription

Reply:
Don't think so (looking at current protobuf docs).  The function only takes a single arg so isn't itemised.